### PR TITLE
WIP: apis/query: add table row responses

### PR DESCRIPTION
**Pull request by sttts**
To support `up get managed`-like requests we have to support CRD columns. With a cross-controlplane API, it's a little bit more tricky because columns can differ.

### CODE REVIEW

File: apis/query/v1alpha1/queryresponse_types.go [hunk 1]
```diff
@@ -77,6 +87,10 @@ type QueryResponseObject struct {
 	// controlPlane is the name and namespace of the object.
 	ControlPlane *QueryResponseControlPlane `json:"controlPlane,omitempty"`
 
+	// tableRow is the object as table row.
+	// +listType=atomic
+	TableRow *metav1.TableRow `json:"tableRow"`
```
Comment (phisco) [inline apis/query/v1alpha1/queryresponse_types.go:92]: My only doubt is whether we should return multiple tables each with their own columns or a single table only with the common columns

**Comment by phisco** [comment]
looking good now, I would poc the implementation before committing to it though

**Comment by sttts** [comment]
> 
looking good now, I would poc the implementation before committing to it though

Right. That was the plan :)

**Review by phisco — APPROVED** [review]

**Comment by sttts** [comment]
This was technically still WIP 😀 But all good.

**Comment by phisco** [comment]
Oops, sorry, wasn't in draft and I didn't notice the title. My bad.

**Pull request merged**